app/services/EmploymentSessionService.scala

```diff
@@ -542,7 +542,7 @@ class EmploymentSessionService @Inject()(employmentUserDataRepository: Employmen
   }
 
   def clear(user: User, taxYear: Int, employmentId: String)
-           (implicit hc: HeaderCarrier, request: AuthorisationRequest[_]): Future[Either[Unit, Unit]] = {
+           (implicit hc: HeaderCarrier, request: CommonAuthorisationRequest): Future[Either[Unit, Unit]] = {
```

Comment:
I presume only Request[_] is enough. Getting the request altogether will be harder here as we need to refactor the whole class.